src/cljs/athens/views/block_page.cljs

```diff
@@ -87,20 +96,23 @@
          [:span {:style {:color "gray"}}
           [breadcrumbs-list {:style {:font-size "1.2rem"}}
            (doall
-             (for [{:keys [node/title block/uid block/string]} parents]
-               ^{:key uid}
-               [breadcrumb {:key (str "breadcrumb-" uid) :on-click #(navigate-uid uid)}
-                (or title string)]))]]
+            (for [{:keys [node/title block/uid block/string]} parents]
+              ^{:key uid}
+              [breadcrumb {:key (str "breadcrumb-" uid) :on-click #(navigate-uid uid)}
+               (or title string)]))]]
```

Comment:
Make sure you run `./script/lint` and `./script/style` when your PR's are ready for review.

Reply:
👍🏼 Yeah these whitespace changes from `cider-fomat-buffer` TODO: adopt athens style.. do we use zprint ? May be worth looking into https://github.com/kkinnear/zprint/blob/master/doc/using/team.md